src/intersperse.js

```diff
@@ -0,0 +1,31 @@
+var _checkForMethod = require('./internal/_checkForMethod');
+var _curry2 = require('./internal/_curry2');
+
+
+/**
+ * Creates a new list with the separator interposed between elements.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig * -> [a] -> [a]
```

Comment:
hmmm. really, the input array could be `[*]` and the return could be `[*]`, so the whole sig probably should be `* -> [*] -> [*]`.